mandatory/3-playing-computer.js

```diff
@@ -9,10 +9,10 @@
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  4. How many times is "f1" called? 2 time
+  5. How many times is "f2" called? 2 time
```

Comment:
Try having another go at counting this

Reply:
I think only two times , when i=2 and i=4 .because the condition is i<5 so it is not going for another round. can explain it please.

Reply:
Hey, there's also i = 0 as well!